src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -222,6 +224,21 @@ private void GetTessellationDataFromGraphicItem(Guid outputPortId, IGraphicItem
                 {
                     CreatePlaneTessellation(package, plane);
                 }
+                else if (graphicItem is IInstanceableGraphicItem instanceableItem &&
+                    instanceableItem.InstanceInfoAvailable 
+                    && package is IInstancingRenderPackage instancingPackage
```

Comment:
@mjkkirschner  Isn't this already cast to `IInstancingRenderPackage` on line 214.  I think we use it in the`if` below but then use `instancingPackage` to add the data.